apps/counter2/settings.js

```diff
@@ -1,20 +1,20 @@
 (function (back) {
     var FILE = "counter2.json";
-    defaults={
+    const defaults={
         counter0:12,
         counter1:0,
         max0:12,
         max1:0,
         buzz: true,
         colortext: true,
     };
-    settings = Object.assign(defaults, require('Storage').readJSON(FILE, true) || {});
+    let settings = Object.assign(defaults, require('Storage').readJSON(FILE, true) || {});
```

Comment:
This can be `const` too, in fact - we don't change what `settings` references :)